### PR TITLE
Update Hackster icon

### PR DESCRIPTION
**Pull request by PeterShaggyNoble**
Hackster

**Issue:** #5598 (not gonna close it yet, pending the outcome of #5614)
**Alexa rank:** [~23.6k](https://www.alexa.com/siteinfo/hackster.io)

### Checklist
  - [ ] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Based on `Hackster Mark\AI\Hackster-Mark_BW.ai` in the [Dropbox folder](https://www.dropbox.com/sh/miwy176rz08ugut/AACm1coV-cxYjJS2xWl2F56ga) linked from [source](https://www.hackster.io/branding)

### CODE REVIEW

**Comment by adamrusted** [comment]
Did you want this one reviewed & merged before we look at introducing the variants, @PeterShaggyNoble?

**Comment by PeterShaggyNoble** [comment]
Yep, the discussion on variants shouldn't be blocking this one as this is their primary icon.

**Review by adamrusted — APPROVED** [review]
LGTM! Will wait on feedback from @service-paradis though before merging, given you've asked for their review!

**Pull request merged**
